app/src/modules/files/routes/add-new.vue

```diff
@@ -23,7 +23,13 @@ function close() {
 		<v-card>
 			<v-card-title>{{ t('add_file') }}</v-card-title>
 			<v-card-text>
-				<v-upload :preset="props.folder ? { folder: props.folder } : undefined" multiple from-url @input="close" />
+				<v-upload
+					:preset="props.folder ? { folder: props.folder } : undefined"
```

Comment:
Obsolete with usage of `folder` props
```suggestion
```

Reply:
Just did a search for all `v-uploads` and youre right. Seems this is the only place where we use the preset so might as well remove it.